package-lock.json

```diff
@@ -20306,28 +20306,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
I wonder why `package-lock.json` is updated when there is no `package.json` change.
"resolved" should point to URI.

Reply:
Interesting ,I didn't catch this. I fixed it to match develop and I'm not sure why `npm install` is updating this either. I'll be more careful going forward.